rudder-core/src/main/resources/hooks.d/policy-generation-finished/50-sigup-cf-serverd.hook

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Parameters are:
+# - 1: generation datetime: ISO-8601 date/time that identify that policy generation. 
+# - 2: generation end time: ISO-8601 date/time when the generation ended (minus these hooks)
+# - 3: space separated list of node id updated during the process
+
+# Signal to cf-serverd that it shall look-up the new promises
```

Comment:
As they are implemented, all hooks are executed for any modification. So as it stands, it is executed every time. I believe it was what was done previously, to.